### PR TITLE
Jump to letter: Enable "#" to jump to the beginning of a list

### PR DESCRIPTION
**Pull request by sualfred**
Currently the "#" does nothing. 
But since 0-9 are always at the beginning it can be mapped with "action(firstpage)"

### CODE REVIEW

**Comment by Guilouz** [comment]
Your changes not working on Kodi 18, when I choose a letter, cursor jump in each letters from sms. Example: when I choose 'Q' letter, cursor jump to letters 'P', 'Q', 'R' and 'S' and that's all.

Like that it's better:

`jumpcmd = ""
        if letter in ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]:
            jumpcmd = "firstpage"
        elif letter in ["A", "B", "C"]:
            jumpcmd = "2"
        elif letter in ["D", "E", "F"]:
            jumpcmd = "3"
        elif letter in ["G", "H", "I"]:
            jumpcmd = "4"
        elif letter in ["J", "K", "L"]:
            jumpcmd = "5"
        elif letter in ["M", "N", "O"]:
            jumpcmd = "6"
        elif letter in ["P", "Q", "R", "S"]:
            jumpcmd = "7"
        elif letter in ["T", "U", "V"]:
            jumpcmd = "8"
        elif letter in ["W", "X", "Y", "Z"]:
            jumpcmd = "9"
        if jumpcmd:
            xbmc.executebuiltin("SetFocus(50)")
            for i in range(40):
                xbmc.executeJSONRPC('{ "jsonrpc": "2.0", "method": "Input.ExecuteAction",\
                    "params": { "action": "jumpsms%s" }, "id": 1 }' % (jumpcmd))
                xbmc.sleep(50)
                if xbmc.getInfoLabel("ListItem.Sortletter").upper() == letter:
                    break`

**Comment by marcelveldt** [comment]
What's the status of this ?

**Comment by sualfred** [comment]
Haven't found some time to look at it again. Please keep it open and don't
merge it yet.

Marcel van der Veldt <notifications@github.com> schrieb am Fr., 30. März
2018, 13:58:

File: resources/lib/plugin_content.py [hunk 1]
```diff
@@ -335,27 +335,29 @@ def alphabetletter(self):
...
+        elif letter in ["M", "N", "O"]:
+            jumpcmd = "jumpsms6"
+        elif letter in ["P", "Q", "R", "S"]:
+            jumpcmd = "jumpsms7"
+        elif letter in ["T", "U", "V"]:
+            jumpcmd = "jumpsms8"
+        elif letter in ["W", "X", "Y", "Z"]:
+            jumpcmd = "jumpsms9"
         if jumpcmd:
             xbmc.executebuiltin("SetFocus(50)")
```
Comment (Angelinas1) [inline resources/lib/plugin_content.py]: @sualfred Work for me in Leia but need to move down in line 360, to prevent blinking as posted @Guilouz

**Comment by Guilouz** [comment]
With latest nightly build, changes from @sualfred are working without issue. Not needed to move line that @Angelinas1 has mentioned.

**Review by Angelinas1 — COMMENTED** [review]
True-----This was Kodi isue...no need to change

**Pull request merged**